### DISCUSSION TITLE
Remaking Gothic on OpenGothic

### DISCUSSION THREAD

**Discussion by Cragann** [Try/OpenGothic#495] [Q&A] [answered]
Hi.
I want to enhance graphics of the original game, but ZenGine is not gonna handle it.
Is OpenGothic capable of rendering larger amounts of polys?
And how easy it would be to make a project based on OpenGothic? Can I make it like a normal Gothic mod or maybe some other approach would be better?

**Comment by Try** [comment, marked as answer]
Hi, @Cragann !
> Is OpenGothic capable of rendering larger amounts of polys?

If you have good enough hardware that can run it - answer should be yes. 'Should' because it's not something we can really test now, so far most advance content was H'Liver and TheChroniclesOfMyrtana - both run well, performance wise.

> Can I make it like a normal Gothic mod 

Yes, OpenGothic basically takes script(from base-game or from mod makes no difference) and runs it. What won't work is: Union as well as any custom *.dll, and Ikarus support is still in progress.

**Reply by Cragann** [reply]
Great to hear that, thank you for answering. Also, what is the potential of running OpenGothic on Android?

**Reply by Try** [reply]
> Also, what is the potential of running OpenGothic on Android?

There is no support for native-activity/android-event loop in engine. Not to mention that Android is quite hostile OS for developer ;)

**Reply by Cragann** [reply]
Ok, fortunately some windows emulators are being developed, so we should be able to play Gothic on mobile within few years. I'll let you know how OG is performing under heavier load.

**Comment by Nindaleth** [comment]
If you'd like to try with an initial testing location (which I suppose will result in the game world staying the same except that one part getting better quality) we can definitely check the resulting performance on our hardware if it helps.

If you don't mind me asking, what do you plan to make? More detailed models and world mesh? Better textures? Animations? Filling the game world with more stuff that the original could not afford? Other improvements aside of graphics-related things? Keeping it all as same as possible, improving the looks via more polygons? Just being curious what to look forward.
